configuration.go

```diff
@@ -149,6 +149,18 @@ func loadOrganizationWhitelist() mapset.Set {
 	return whitelist
 }
 
+func getAPISpecFile() (string, error) {
+	specFile := serverCfg.GetString("api_spec_file")
+
+	fileInfo, err := os.Stat(specFile)
+	if os.IsNotExist(err) {
```

Comment:
prob. might be good to add a test & !os.IsDir() or & os.IsRegular()

Reply:
added, also added another check in test.sh